### PR TITLE
Merge config() and config_list() for Juniper.

### PR DESCRIPTION
**Pull request by jeffkala**
Deprecation warning for config_list() and repoint it to config() which now turns str into list, and runs commands through for loop to load them into config.

### CODE REVIEW

File: pyntc/devices/jnpr_device.py [hunk 1]
```diff
@@ -116,22 +117,36 @@ def close(self):
...
+        """
+        if isinstance(commands, str):
+            commands = [commands]
         try:
             for command in commands:
                 self.cu.load(command, format=format)
-
             self.cu.commit()
         except ConfigLoadError as e:
             raise CommandListError(commands, command, e.message)
```
Comment (jmcgill298) [inline pyntc/devices/jnpr_device.py:137]: This should check to see if `commands` was passed as a list or a string, and then raise either `CommandError` or `CommandListError` as appropriate.

**Review by jmcgill298 — CHANGES_REQUESTED** [review]
Thanks again for the PR @jeffkala There is one comment on updates, and would also include the same updates to test cases as #195

**Comment by jeffkala** [comment]
@jmcgill298, thanks for the review. Will implement these soon.

**Review by jmcgill298 — CHANGES_REQUESTED** [review]

**Review by jmcgill298 — APPROVED** [review]

**Comment by jmcgill298** [comment]
Thanks @jeffkala I made one change to how the config_as_list test case was validating the calls.

**Pull request merged**